app/decorators/manageiq/providers/openstack/cloud_manager/template_decorator.rb

```diff
@@ -0,0 +1,9 @@
+class ManageIQ::Providers::Openstack::CloudManager::TemplateDecorator < MiqTemplateDecorator
+  def single_quad
+    {}
+  end
+
+  def size_tooltip
```

Comment:
I think you should give it a little more meaningful name, like `provisioning_volume_size_tooltip` but you might come up with something better.